docs/docs/ref-04-tags-and-attributes.md

```diff
@@ -53,15 +53,15 @@ These standard attributes are supported:
 
 ```
 accept acceptCharset accessKey action allowFullScreen allowTransparency alt
-async autoComplete autoFocus autoPlay cellPadding cellSpacing charSet challenge
-checked classID className colSpan cols content contentEditable contextMenu controls
-coords crossOrigin data dateTime defer dir disabled download draggable encType
-form formAction formEncType formMethod formNoValidate formTarget frameBorder
+async autoComplete autoFocus autoPlay capture cellPadding cellSpacing charSet
+challenge checked classID className cols colSpan content contentEditable contextMenu
+controls coords crossOrigin data dateTime defer dir disabled download draggable
+encType form formAction formEncType formMethod formNoValidate formTarget frameBorder
 headers height hidden high href hrefLang htmlFor httpEquiv icon id inputMode
 keyParams keyType label lang list loop low manifest marginHeight marginWidth max
-maxLength media mediaGroup method min multiple muted name noValidate open
+maxLength media mediaGroup method min minlength multiple muted name noValidate open
```

Comment:
Should be `minLength`. But that's ok, we always update this list on release and we'll do it again.